ebay-frontend/src/app/upcoming-events-screen/upcoming-events-screen.component.scss

```diff
@@ -1,7 +1,57 @@
 .upcoming-card {
+        background-color: #FFFFFF;
+        display: flex;
+        justify-content: space-between;
+       
+
+}
+.left {
+  background: white;
```

Comment:
you shouldn't need to set the background color for left and right if you're setting it for .upcoming-card